tareas/clase-6/tarea-clase-6.js

```diff
@@ -14,3 +14,143 @@ Al hacer click en "calcular", mostrar en un elemento pre-existente el mayor sala
 
 Punto bonus: si hay inputs vacíos, ignorarlos en el cálculo (no contarlos como 0).
 */
+
+const integrantesGrupoFamiliar = Number(prompt("Cuántas persona integran su grupo familiar? "));
+
+for (let i = 1; i <= integrantesGrupoFamiliar; i++) {
+    const nuevoLabel = document.createElement('label');
+    nuevoLabel.textContent = "Ingrese edad del intregante " + i;
+    const br = document.createElement("br");
+    const header = document.querySelector("header");
+    header.appendChild(br);
+    header.appendChild(nuevoLabel);
+
+    const nuevoInput = document.createElement('input');
+    header.appendChild(nuevoInput);
+}
+
+const $botoncalcular = document.querySelector("#calcular");
+
+$botoncalcular.onclick = function () {
+
+    const listadoDeEdades = document.querySelectorAll("input");
+
+
+    let edadMayor = Number(listadoDeEdades[0].value);
+    
+    for (let i = 0; i < listadoDeEdades.length; i++) {
+        if(edadMayor!==""){
+            if (Number(listadoDeEdades[i].value) > edadMayor) {
+                edadMayor = Number(listadoDeEdades[i].value);
+            }
+        }else{
+            continue;
+        }
+    }
+
+    document.querySelector("#edad-mayor").textContent = "La edad del mayor integrante de la familia es " + edadMayor + " años."
+
+    let edadMenor = Number(listadoDeEdades[0].value);
+    for (let i = 0; i < listadoDeEdades.length; i++) {
+        if (Number(listadoDeEdades[i].value) < edadMenor) {
+            edadMenor = Number(listadoDeEdades[i].value);
+        }
+    }
+
+    document.querySelector("#edad-menor").textContent = "La edad del menor integrante de la familia es " + edadMenor + " años."
+
+
+    let acumuladoDeEdad = 0;
+    for (let i = 0; i < listadoDeEdades.length; i++) {
+        acumuladoDeEdad += Number(listadoDeEdades[i].value);
+    }
+    promedioDeEdad = acumuladoDeEdad / listadoDeEdades.length;
+
+    document.querySelector("#edad-promedio").textContent = "La edad promedio de la familia es " + promedioDeEdad + " años."
+
+
+
+    return false
+}
+
+const $botonLimpiar = document.querySelector("#limpiar");
+
+$botonLimpiar.onclick = function () {
+
+    location.reload();
+    return false
+
+}
+
+const $botonagregar = document.querySelector("#agregar");
+$botonagregar.onclick = function () {
+
+    const nuevoLabel = document.createElement('label');
+    nuevoLabel.textContent = "Ingrese salario del intregante "
+    const br = document.createElement("br")
+    const salario = document.querySelector("#salario")
+    salario.appendChild(br);
+    salario.appendChild(nuevoLabel);
+
+    const nuevoInput = document.createElement('input');
```

Comment:
```suggestion
    const nuevoInput = document.createElement("input");
```

Hay que ser consistente con el uso de comillas dobles y simples!